src/Search/Provider/SQLProvider.php

```diff
@@ -461,7 +462,32 @@ public static function getSelectCriteria(string $itemtype, int $ID, bool $meta =
                     return array_merge($SELECT, $ADDITONALFIELDS);
                 }
                 break;
-
+            case "glpi_consumables.id":
+                if (
+                    $itemtype == ConsumableItem::class
+                    && ($ID == 17 || $ID = 19)
+                    && $opt_arrays[$ID]["datatype"] == 'count'
+                ) {
+                    // forces the LEFT JOIN filter directly into the SELECT statement to optimise data loading
+                    if ($ID == 19) {
+                        $SELECT = [
+                            new QueryExpression(
+                                QueryFunction::sum(new QueryExpression("CASE WHEN $table$addtable.date_out IS NULL THEN 1 ELSE 0 END")),
```

Comment:
For clarity I think when named parameters are used, all parameters in that call should use them or none of them.